packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -135,9 +170,14 @@ def test_vip(dcos_api_session, container: Container, vip_net: Network, proxy_net
         raise AssertionError('\n******************************************\n'.join(failure_stack))
 
 
-def setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net):
+def setup_vip_workload_tests(dcos_api_session, pod, container, vip_net, proxy_net):
```

Comment:
How about changing the signature of this to
```
setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net,  uses_pod=False)
```
* We could use some other term for `uses_pod`.
* But, a keyword arg with the default of `False` might be desirable.

With this, the existing tests will remain unchanged, and the new test could send an flag with `=True`.